### ISSUE TITLE
Does not work on GoDaddy

### ISSUE THREAD

**Issue by homestar9** [jswanner/DontF-WithPaste#120]
I love this extension, and for the most part, it works well. However, I ran into an issue where it doesn't work on https://sso.godaddy.com/,  That's where GoDaddy sends you a 2-step verification code.  I generally copy the code from my 2FA device and try to paste it in, but GoDaddy blocks it.

Here's a screenshot
![image](https://user-images.githubusercontent.com/5341142/182037136-c846b3e6-d4f1-40d8-ac82-6e8143f4c7e8.png)

Pattern:
![image](https://user-images.githubusercontent.com/5341142/182037143-043b6eeb-79bf-45b8-855e-c49ce5f98e0d.png)

**Comment by jswanner** [comment]
@homestar9, I'm not able to reproduce this issue, although I just realized you were on the text message verification form and I used the email verification & TOTP forms. I didn't experience any blocking of paste on any of their forms. Maybe they're only blocking the Windows/Linux keyboard shortcut for paste (control + v) and nothing else. Can you try pasting from the context menu (right click) or from the browser's Edit menu?

**Comment by homestar9** [comment]
Thanks for the quick response @jswanner.  I also tried the context menu, and Godaddy still blocked it.  I have some good news, though. I was trying to record a GIF of the problem occurring, and while setting up my test, I made sure to have Don't F pinned to the browser bar.  When I had the plugin pinned, the problem went away, and I could paste into the form field both by using the context menu and CTRL+V.  

I'll close the issue and play around with it some more.  If I can accurately reproduce the problem again, I'll create a new ticket with some more details.
